clients/typescript/src/types.ts

```diff
@@ -69,6 +139,43 @@ interface PathfinderUniqueParams {
   num_threads: number;
 }
 
+/**
+ * @typedef {Object} PathfinderParams
+ * Parameters for the Pathfinder algorithm.
+ * @property {string | StanVariableInputs} [data=""] The data for the model.
+ * @property {number} [num_paths=4] The number of individual paths to run.
+ * @property {string | StanVariableInputs | string[] | StanVariableInputs[]} [inits=""]
+ * The initial values for the algorithm. If an array, must have length `num_paths`.
+ * @property {number | null} [seed] The seed for the random number generator.
+ * If unspecified, a random seed will be generated.
+ * @property {number} [id=1] The ID for the first path.
+ * @property {number} [init_radius=2.0] Radius to initialize unspecified parameters within.
+ * The parameter values are drawn uniformly from the interval
+ * `[-init_radius, init_radius]` on the unconstrained scale.
+ * @property {number} [num_draws=1000] The number of draws to take for each path.
+ * @property {number} [max_history_size=5] History size used by the internal L-BFGS algorithm
+ * to approximate the Hessian.
+ * @property {number} [init_alpha=0.001] Initial step size for the internal L-BFGS algorithm.
+ * @property {number} [tol_obj=1e-12] Convergence tolerance for the objective function.
+ * @property {number} [tol_rel_obj=1e4] Relative convergence tolerance for the objective function.
+ * @property {number} [tol_grad=1e-8] Convergence tolerance for the gradient norm.
+ * @property {number} [tol_rel_grad=1e7] Relative convergence tolerance for the gradient norm.
+ * @property {number} [tol_param=1e-8] Convergence tolerance for the changes in parameters.
```

Comment:
Is this absolute? Seems like scale would be an issue?

Reply:
Yes, and, uh, *shrug*